wordpress/wp-content/plugins/sil-dictionary-webonary/webonary/Webonary_Utility.php

```diff
@@ -3,6 +3,12 @@
 
 class Webonary_Utility
 {
+	// TODO: Check if this is reasonable, or should be increased.
+	private const DEFAULT_POSTS_PER_PAGE = 25;
```

Comment:
Unfortunately, Webonary is still using PHP 7.0, so this results in an error:

`PHP Parse error:  syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE) `

Perhaps a define() or private static?